stock_voucher/models/stock_picking.py

```diff
@@ -66,66 +65,60 @@ def get_estimated_number_of_pages(self):
         if lines_per_voucher == 0:
             estimated_number_of_pages = 1
         else:
-            operations = len(self.pack_operation_ids)
+            operations = len(self.move_lines)
             estimated_number_of_pages = int(ceil(
                 float(operations) / float(lines_per_voucher)
             ))
         return estimated_number_of_pages
 
-    @api.one
     @api.constrains('picking_type_id')
     @api.onchange('picking_type_id')
     def _get_book(self):
         self.book_id = self.picking_type_id.book_id
 
-    @api.multi
-    @api.onchange(
-        'pack_operation_ids',
-        'pack_operation_product_ids',
-        'pack_operation_pack_ids',
-    )
+    @api.constrains('move_lines')
+    @api.onchange('move_lines')
     def product_onchange(self):
         # self.declared_value = 0
+        SaleOrderLine = self.env['sale.order.line']
         for rec in self:
             if not rec.automatic_declare_value:
                 return False
             done_value = 0.0
             picking_value = 0.0
             inmediate_transfer = True
             pricelist = False
-            for x in rec.pack_operation_ids:
-                order_line = rec.env['sale.order.line'].search(
+            for x in rec.move_lines:
```

Comment:
done use x as var to identify the iteration, use a var name at least of 3 letters. this is a pylint complaint. you could use `line` , `move_line`, `item`, etc